From: Teemu Ikonen <tpikonen@gmail.com>
Subject: Set zero page size to GtkSpinButtons in export dialog.
X-git-branch: p/spinbuttons

    Fixes deprecation warning from GTK.

    Also re-save the sizechooser.glade file in glade, causing
    some reordering in the XML file.


 src/sizechooser.glade |   29 +++++++++++++++++++----------
 1 files changed, 19 insertions(+), 10 deletions(-)

diff --git a/src/sizechooser.glade b/src/sizechooser.glade
index 77b3d99..da8bdbe 100644
--- a/src/sizechooser.glade
+++ b/src/sizechooser.glade
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE glade-interface SYSTEM "glade-2.0.dtd">
-<!--Generated with glade3 3.2.2 on Wed Sep 19 00:07:16 2007 by tpikonen@slobo-->
+<?xml version="1.0"?>
 <glade-interface>
+  <!-- interface-requires gtk+ 2.6 -->
+  <!-- interface-naming-policy toplevel-contextual -->
   <widget class="GtkWindow" id="window1">
     <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
     <child>
@@ -15,20 +15,25 @@
             <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
             <child>
               <widget class="GtkRadioButton" id="size_current">
+                <property name="label" translatable="yes">Current size</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
-                <property name="label" translatable="yes">Current size</property>
                 <property name="active">True</property>
                 <property name="draw_indicator">True</property>
               </widget>
+              <packing>
+                <property name="position">0</property>
+              </packing>
             </child>
             <child>
               <widget class="GtkRadioButton" id="size_640">
+                <property name="label" translatable="yes">640 x 480</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
-                <property name="label" translatable="yes">640 x 480</property>
                 <property name="draw_indicator">True</property>
                 <property name="group">size_current</property>
                 <signal name="toggled" handler="size_toggled"/>
@@ -39,10 +44,11 @@
             </child>
             <child>
               <widget class="GtkRadioButton" id="size_1024">
+                <property name="label" translatable="yes">1024 x 768</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
-                <property name="label" translatable="yes">1024 x 768</property>
                 <property name="draw_indicator">True</property>
                 <property name="group">size_current</property>
               </widget>
@@ -52,10 +58,11 @@
             </child>
             <child>
               <widget class="GtkRadioButton" id="size_1600">
+                <property name="label" translatable="yes">1600 x 1200</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
-                <property name="label" translatable="yes">1600 x 1200</property>
                 <property name="draw_indicator">True</property>
                 <property name="group">size_current</property>
               </widget>
@@ -69,16 +76,18 @@
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
                 <child>
                   <widget class="GtkRadioButton" id="size_custom">
+                    <property name="label" translatable="yes">Custom: </property>
                     <property name="visible">True</property>
                     <property name="can_focus">True</property>
+                    <property name="receives_default">False</property>
                     <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
-                    <property name="label" translatable="yes">Custom: </property>
                     <property name="draw_indicator">True</property>
                     <property name="group">size_current</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
                     <property name="fill">False</property>
+                    <property name="position">0</property>
                   </packing>
                 </child>
                 <child>
@@ -87,7 +96,7 @@
                     <property name="can_focus">True</property>
                     <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
                     <property name="max_length">5</property>
-                    <property name="adjustment">1920 1 65536 1 100 10</property>
+                    <property name="adjustment">1920 1 65536 1 100 0</property>
                     <property name="snap_to_ticks">True</property>
                     <property name="numeric">True</property>
                     <signal name="value_changed" handler="sizespin_cb"/>
@@ -114,7 +123,7 @@
                     <property name="can_focus">True</property>
                     <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
                     <property name="max_length">5</property>
-                    <property name="adjustment">1440 1 65536 1 100 10</property>
+                    <property name="adjustment">1440 1 65536 1 100 0</property>
                     <property name="numeric">True</property>
                     <signal name="value_changed" handler="sizespin_cb"/>
                   </widget>
